web/views.py

```
@@ -1301,29 +1301,57 @@
+
+                # Get the status from the form
+                status = value
+
+                # Update the attendance record
                 attendance, created = SessionAttendance.objects.update_or_create(
-                    session=session, student=student, defaults={"status": status}
+                    session=session,
+                    student=student,
+                    defaults={
+                        "status": status,
+                        "verified_by_teacher": True if status in ["present", "late", "excused"] else False,
```

Ruff (0.8.2)

Remove unnecessary `True if ... else False`

Remove unnecessary `True if ... else False`

(SIM210)